prov/tcp/src/tcpx_attr.c

```diff
@@ -63,7 +63,11 @@ static struct fi_ep_attr tcpx_ep_attr = {
 	.protocol_version = 0,
 	.max_msg_size = SIZE_MAX,
 	.tx_ctx_cnt = 1,
-	.rx_ctx_cnt = 1
+	.rx_ctx_cnt = 1,
+	.max_order_raw_size = -1,
+	.max_order_war_size = -1,
+	.max_order_waw_size = -1,
```

Comment:
Use SIZE_MAX.  Also, I don't think we support arbitrary WAR sizes.  We don't stall writes until all data from a prior read has been sent back to the requestor.  WAR may not be valid at all, depending on how read requests are handled.